docs/vertiq_testing_tool/vertiq_testing_tool_start_guide.rst

```diff
@@ -0,0 +1,142 @@
+
+In order to visualize the module's drive voltage, navigate to the widget labeled 'Select a metric to plot'. The upper dropdown menu allows you to pick a client, and the lower a client entry.
+Please refer to :ref:`Getting Started with Vertiq's APIs <getting_started_with_apis>` to learn more about IQUART clients.
+
+In the upper dropdown menu, select *brushless_drive*. This corresponds to the :ref:`Brushless Drive <brushless_drive>` client.
+In the lower dropdown menu, select *drive_volts*, which corresponds with brushless drive's *drive_volts* entry.
+Now, click 'Update Plot'. 
+
+Notice that the *drive_volts* metric is added to the 'Active metrics' widget. By default, *obs_velocity*, *watts*, and *amps* are preloaded in the 'Active metrics' widget.
+Selecting a metric activates it, and updates the live plot with the now active metric. Only one metric can be active at a time, but there is no limit to the number of metrics that can be added to the 'Active metrics' widget.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/remove_metric_example.png
+To remove a metric from the 'Active metrics' widget, simply right click on it and select 'Remove metric'.
+
+
+===============================
+Auto Command and Ramp to Target
+===============================
+.. image:: ../_static/vertiq_testing_tool_pictures/voltage_input.png
+
+Another useful feature in the Voltage, Velocity, and PWM Input tabs is the 'Ramp to target' option found in the Control Panel.
+This feature is enabled by default, which means when you click 'Send Command' after setting the desired Target Voltage, the module ramps to the target voltage incrementally over the time defined in Ramp Time. 
+The increments are determined by the ramp time, so increasing the ramp time will make it so that the module takes longer to reach the target voltage.
+
+The 'Auto Command' feature is disabled by default. When enabled, you will not have to click the 'Send Command' button to send a voltage command. 
+Instead, the application will automatically send the command whenever the target voltage value changes.
+
+Be very cautious when 'Auto Command' is enabled and 'Ramp to Target' is disabled. It is recommended to lower the 'Step Amount' to avoid large jumps between each target voltage command.
+
+
+.. image:: ../_static/vertiq_testing_tool_pictures/voltage_input_ramp_example.png
+
+The example above shows the plot of the observed velocity of the module. 'Ramp to target' was enabled when the module was commanded to 5 Volts. The plot shows that the module took 5 seconds to ramp up to the target voltage.
+The module was then commanded to 3 Volts, and then 0 Volts.
+
+
+.. image:: ../_static/vertiq_testing_tool_pictures/voltage_input_ramp_disabled_example.png
+
+In this example, 'Ramp to target' was disabled when the module was commanded to 1 Volt, 2 Volts, 3 Volts, and finally 0 Volts. The steep jumps between each target voltage command shows that the module was not ramping.
```

Comment:
Here and on line 132, you could probably just say 1V, 2V, 3V, instead of writing out volts. That's the generally accepted unit abbreviation for volts